moveit_planners/test_configs/prbt_moveit_config/launch/demo.launch.py

```diff
@@ -59,13 +59,13 @@ def generate_launch_description():
     joint_limits_yaml = load_yaml(
         "moveit_resources_prbt_moveit_config", "config/joint_limits.yaml"
     )
-    cartesian_limits_yaml = load_yaml(
-        "moveit_resources_prbt_moveit_config", "config/cartesian_limits.yaml"
+    pilz_cartesian_limits_yaml = load_yaml(
```

Comment:
should we do anything to try and migrate the existing file to the new filename for users who are updating their configuration? (it's possible there are very few users who fall in this bucket - we haven't even cut a release with MSA yet AFAIK).

Reply:
Probably should print a warning from `moveit_configs_builder.py`, huh?

Reply:
Added a print statement that gives output like this:

```
[INFO] [launch]: All log files can be found below /home/andy/.ros/log/2022-07-08-13-02-04-616594-andy-G7-7500-225832
[INFO] [launch]: Default logging verbosity is set to INFO
WARNING:root:cartesian_limits.yaml is deprecated, please rename to pilz_cartesian_limits.yaml
[ERROR] [launch]: Caught exception in launch (see debug for traceback): Caught exception when trying to load file of format [py]: "File /home/andy/ws_ros2/install/moveit_resources_panda_moveit_config/share/moveit_resources_panda_moveit_config/config/pilz_cartesian_limits.yaml doesn't exist"
```